package.json

```diff
@@ -2,25 +2,24 @@
   "name": "@npm/minify-registry-metadata",
   "description": "minimal regsitry packuments. :corgi:",
   "version": "1.0.2",
-  "bugs": {
-    "url": "https://github.com/soldair/minify-registry-metadata/issues"
-  },
   "devDependencies": {
     "changes-stream": "^2.2.0",
     "concurrent-couch-follower": "^1.2.0",
     "normalize-registry-metadata": "^1.1.2",
     "request": "^2.80.0",
-    "tape": "^4.6.3"
+    "tap": "^14.6.1"
   },
-  "homepage": "https://github.com/soldair/minify-registry-metadata#readme",
-  "keywords": [],
   "license": "ISC",
   "main": "index.js",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/soldair/minify-registry-metadata.git"
+    "url": "git+https://github.com/npm/minify-registry-metadata.git"
   },
   "scripts": {
-    "test": "tape test/*.js"
+    "test": "tap",
+    "snap": "tap"
```

Comment:
curious: what is the `snap` script going to be used for? is it some kind of convention to indicate the tests are using snapshot? 🤔

Reply:
Yeah I'm curious too @isaacs. I'm not sure what's happening here either.

Reply:
Node-tap (as of v14) looks at the npm lifecycle script environment variable to see if it should check or re-generate snapshots. So, running `npm run snap` here will run the tests in snapshot generation mode, like doing `tap --snapshot`. 

More info about snapshots: https://node-tap.org/docs/api/snapshot-testing/